### PR TITLE
Positions

### PR DESCRIPTION
**Pull request by qjd2413**

### CODE REVIEW

File: app/controllers/db.js [hunk 1]
```diff
@@ -23,6 +36,40 @@ module.exports = {
         fields: ['pin']
       }
     );
+  },
+  userAuthorization: function(id) {
+    return Brother.findById(id, {
+      include: [SysAdmin, Position]
```
Comment (kalebdavis) [inline app/controllers/db.js]: Can you point me to documentation around what this does? The sequelize docs are very hard to search through.

File: app/controllers/db.js [hunk 2]
```diff
@@ -23,6 +36,40 @@ module.exports = {
...
+    }).then(function(brother) {
+      if(brother.SysAdmin) {
+        return 'sysadmin';
+      };
+      if(brother.Positions.length) {
+        var positions = lodash.uniq(lodash.map(brother.Positions, 'dataValues.AuthorizationId'));
+        return q.all(lodash.map(positions, function(positionId) {
+          console.log(positionId);
```
Comment (kalebingerick) [inline app/controllers/db.js]: should use logger if important or be taken out

File: app/controllers/db.js [hunk 3]
```diff
@@ -23,6 +36,40 @@ module.exports = {
...
+          return Authorization.findById(positionId);
+        })).then(function(authorizations) {
+          authorizations = lodash.map(authorizations, 'dataValues');
+          var maxAuth = lodash.minBy(authorizations, 'level');
+          return maxAuth.name;
+          console.log(maxAuth);
```
Comment (kalebingerick) [inline app/controllers/db.js]: same with logger as above

File: app/controllers/db.js [hunk 4]
```diff
@@ -23,6 +36,40 @@ module.exports = {
...
+      if(brother.SysAdmin) {
+        return 'sysadmin';
+      };
+      if(brother.Positions.length) {
+        var positions = lodash.uniq(lodash.map(brother.Positions, 'dataValues.AuthorizationId'));
+        return q.all(lodash.map(positions, function(positionId) {
+          console.log(positionId);
+          return Authorization.findById(positionId);
+        })).then(function(authorizations) {
+          authorizations = lodash.map(authorizations, 'dataValues');
```
Comment (kalebdavis) [inline app/controllers/db.js]: What are dataValues and level here? There don't seem to correspond to anything that I can see, help a noob out.

File: public/app/services/positions.service.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
+
+(function() {
+
+  var positionService = function($http) {
+    
+    var getHttp = function(url) { return $http.get(url)
```
Comment (kalebingerick) [inline public/app/services/positions.service.js]: move the return to another line

**Comment by kalebingerick** [comment]
get rid of logging statements.

**Pull request merged**